Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -784,7 +787,7 @@ public async Task OpenDropdown()
             if ( HasFilteredData && AutoPreSelect )
             {
                 ActiveItemIndex = 0;
-                ExecuteAfterRender( () => ScrollItemIntoView( Math.Max( 0, ActiveItemIndex ) ) );
+                ExecuteAfterRender( () => ScrollItemIntoView( ActiveItemIndex ) );
```

Comment:
I think `Math.Max( 0, ActiveItemIndex )` is more correct. Because when `ActiveItemIndex = -1` then `0` should be the max value used for scroll.

Reply:
here I assume it will be 0 because of line 789 "ActiveItemIndex = 0"

Reply:
Never assume anything when programming 😅

Reply:
I have to assume at least that the computer will do what it's told :)

Reply:
here's an example ;) https://docs.microsoft.com/en-us/cpp/assembler/masm/assume?view=msvc-170

Reply:
Who knows, maybe `ActiveItemIndex` in the future changes, and has logic in it... then your assignment might not be = 0. :)
Altough it most likelly always be haha